### PR TITLE
[DE] Add `Beatmap/Stack leniency`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Stack_leniency/de.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Im [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) im [Beatmap-Editor](/wiki/Client/Beatmap_editor), unter dem Tab `Advanced`, kann die Stackzuordnung eingestellt werden. Die dargestellten Werte im Editor reichen von `0`bis `10` und entsprechen dem Bereich `0,0` bis `1,0` des Parameters `StackLeniency` in den [`.osu`-Dateien](/wiki/Client/File_formats/osu_(file_format)).
+
+## Verhalten
+
+![Das Stapeln von Hit-Objekten während der Bearbeitung kann dazu führen, dass sie sich überlappen.](img/stacking-editor.jpg "Gestapelte Hit-Objekte beim Editieren")
+
+![Während des Spielens werden die Objekte jedoch schräg neben einander gelegt.](img/stacking-playtest.jpg "Gestapelte Hit-Objekte im Spiel")
+
+Die Stackzuordnung steuert zusammen mit der [Approach-Rate](/wiki/Beatmap/Approach_rate), welche Circles und Sliderteile einem Stack zugeordnet werden.[^stacking-algorithm] Der Stack besteht aus Objekten, die nicht weiter als `preempt * StackLeniency` Millisekunden voneinander entfernt sind, wobei `preempt` die [Breite des Einblendungsfensters](/wiki/Beatmap/Approach_rate#timing-der-animationen) und `StackLeniency` der Wert aus der `.osu`-Datei der Beatmap ist.
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Stack_leniency/de.md:28]: I had the idea to call approach rate `Einblendrate` or `Einblendungsrate` as an alternative.

Though that's for another time.

File: wiki/Beatmap/Stack_leniency/de.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - automated stacking
+  - autostacking
+  - stapeln
+---
+
+# Stackzuordnung
+
+**Stackzuordnung** (auch *stack leniency* genannt) ist ein [Beatmap-Parameter](/wiki/Beatmap), der steuert, welche [Circles](/wiki/Gameplay/Hit_object/Hit_circle) und [Sliderteile](/wiki/Gameplay/Hit_object/Slider) im [osu!-Spielmodus](/wiki/Game_mode/osu!) automatisch in einem [Stack](/wiki/Beatmapping/Mapping_techniques/Stack) gruppiert werden. Ein kleinerer Wert reduziert die maximale Zeitspanne, nach der [Hit-Objekte](/wiki/Gameplay/Hit_object) nicht mehr Teil desselben Stacks sind. Umgekehrt werden bei einem höheren Wert Objekte gruppiert, die zeitlich weiter auseinander liegen.
+
+Im [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) im [Beatmap-Editor](/wiki/Client/Beatmap_editor), unter dem Tab `Advanced`, kann die Stackzuordnung eingestellt werden. Die dargestellten Werte im Editor reichen von `0`bis `10` und entsprechen dem Bereich `0,0` bis `1,0` des Parameters `StackLeniency` in den [`.osu`-Dateien](/wiki/Client/File_formats/osu_(file_format)).
```
Comment (FreezyLemon) [inline wiki/Beatmap/Stack_leniency/de.md:20]:
```suggestion
Im [Song Setup](/wiki/Client/Beatmap_editor/Song_setup) im [Beatmap-Editor](/wiki/Client/Beatmap_editor), unter dem Tab `Advanced`, kann die Stackzuordnung eingestellt werden. Die dargestellten Werte im Editor reichen von `0` bis `10` und entsprechen dem Bereich `0,0` bis `1,0` des Parameters `StackLeniency` in den [`.osu`-Dateien](/wiki/Client/File_formats/osu_(file_format)).
```

**Review by FreezyLemon — COMMENTED** [review]
`Stapel` and `Hit-Objekte stapeln` just feels wrong, and I'm not sure I ever heard anyone say it. Can't we introduce `stacken` as a term and use that?

**Comment by The-Last-Cookie** [comment]
> Stapel and Hit-Objekte stapeln just feels wrong, and I'm not sure I ever heard anyone say it. Can't we introduce stacken as a term and use that?

I made a poll on the German discord to know what others think about this and of 3 people there were 3 votes for using the German version.

Regarding the verb usage, I prefer the translation `stapeln` too. For the noun, I'm fine with both.

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**